osclib/conf.py

```diff
@@ -44,6 +44,8 @@
         'openqa': 'https://openqa.opensuse.org',
         'lock': 'openSUSE:%(project)s:Staging',
         'lock-ns': 'openSUSE',
+        'delreq-monitor': 'factory-maintainers',
```

Comment:
I am not clear what is the purpose behind the word `monitor` in the key. Seems more like a `virtual delete request queue` or `backlog` or `pending list`. I understand the list will be monitored to determine when it is appropriate to actually accept the delete, but given this controls the `virtual delete` request feature it seems like that would be more appropriate to be in the key name.

Perhaps `virtual-delete-review` or somesuch as that provides a much better idea of what this is for at a glance.